### ISSUE TITLE
Level: "Invader"

### ISSUE THREAD

**Issue by colonelfazackerley** [andybalaam/rabbit-escape#90]
I was working on this at the same time as @tttppp was working on #89, without realising. Hence the similar names.
```
:name=Invader
:description=Alien beings from a dying planet.
:num_rabbits=9
:num_to_save=8
:block=2
:climb=1
:dig=4
:bridge=2
:bash=2
:explode=2
:rabbit_delay=10
:music=tryad-witness
   Q         
  Q      O   
 Q #     #   
    #   #    
   #######   
  ## ### ##  
 ########### 
 # ####### # 
 # #     # # 
    ## ##    
             
```
The start of this level is quite buttock clenching. Not sure the level is fun.

I have to stop messing on the computer now, and play with the kids. Then it's a race to a pacman themed level.

**Comment by tttppp** [comment]
Iconic level :-)  It's got some tricky timing without being insane.

My solution (in confusing diagram form):
https://cloud.githubusercontent.com/assets/7073263/10411826/889649fe-6f6a-11e5-80b6-9eb5209000ee.png

**Comment by colonelfazackerley** [comment]
That's similar to my solution without being identical.

**Comment by andybalaam** [comment]
Really tricky at the beginning, but I managed it on my phone so must be ok. I solved it without using the climber but can see how it would be useful. Had me scratching my head for several attempts.

**Comment by andybalaam** [comment]
Btw this level is definitely fun.

**Comment by colonelfazackerley** [comment]
merged to master f96dd5696bc0d3c0f44b4856ba75e3a7d2bc201a
